api/v1/ibmblockcsi_types.go

```diff
@@ -112,12 +115,34 @@ type IBMBlockCSINodeSpec struct {
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
 }
 
+// IBMBlockCSICallHomeSpec defines the desired state of IBMBlockCSICallHome
+type IBMBlockCSICallHomeSpec struct {
+	Repository string `json:"repository"`
+	Tag        string `json:"tag"`
+
+	SecretName string `json:"secretName"`
```

Comment:
don't it better to add a secret namespace option? it's hard to add?

Reply:
there no such thing in the api.. [it takes the namespace of the pod](https://kubernetes.io/docs/concepts/configuration/secret/#restrictions)..

Reply:
oh ok my bad